src/interactions/warframe/Builds.js

```
@@ -435,31 +437,31 @@ export default class Builds extends Interaction {
-              style: MessageButtonStyles.PRIMARY,
+              style: ButtonStyle.Primary,
             }),
-            new MessageButton({
+            new ButtonBuilder({
               label: 'Next',
               customId: 'next',
-              style: MessageButtonStyles.SECONDARY,
+              style: ButtonStyle.Secondary,
             }),
           ],
         });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)